src/test/java/seedu/bookmark/testutil/BookBuilder.java

```diff
@@ -33,7 +34,7 @@ public BookBuilder() {
         name = new Name(DEFAULT_NAME);
         genre = new Genre(DEFAULT_GENRE);
         totalPages = new TotalPages(DEFAULT_TOTAL_PAGES);
-        bookmark = new Bookmark();
+        bookmark = new Bookmark(DEFAULT_BOOKMARK);
```

Comment:
the constructor Bookmark() creates a default bookmark with value 0. Perhaps we can discuss which approach we want to take for setting up a Default Bookmark of Pg 0?

Reply:
That is a good point! I did not notice the Bookmark constructor